main.py

```diff
@@ -0,0 +1,55 @@
+import random
+import art
+import os
+from game_data import data
+
+# compare() calculates which choice has more followers and checks if user's answer is that choice. Returns bool
+def compare(choice_a: dict, choice_b: dict) -> bool:
+    user_answer = input('Who has more instagram followers? Type "A" or "B": ').lower()
+    
+    if choice_a['follower_count'] > choice_b['follower_count']:
```

Comment:
See if you can do this in one line:

solution = 'a' if <condition> else 'b'

Reply:
I understand

Reply:
Great